data-protocols/ids/ids-api-multipart-dispatcher-v1/src/test/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/MultipartEndpointDataReferenceRequestSenderTest.java

```diff
@@ -110,10 +114,6 @@ void getResponseContent() throws Exception {
                 .isEqualTo(payload);
     }
 
-    private static DynamicAttributeToken createDatToken() {
-        return new DynamicAttributeTokenBuilder()._tokenValue_(UUID.randomUUID().toString()).build();
-    }
-
```

Comment:
seems as similar issue with rebase as in the other class, this method was moved to the top of the class